website/docs/d/database_user.html.markdown

```diff
@@ -22,7 +22,7 @@ resource "mongodbatlas_database_user" "test" {
 	password      = "test-acc-password"
 	project_id      = "<PROJECT-ID>"
 	database_name = "admin"
-	
+
```

Comment:
Shouldn't we have a SCRAM Authentication example (what is here now) and x509 Certificate example now (e.g. admin and auth)?